ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -261,6 +261,9 @@ def send_schema1_event(self, ev):
         # the AppRegistryNotReady exception
         from ansible_ai_connect.ai.api.utils.segment import send_schema1_event
 
+        # Set the "seated user" flag to ensure prompt/response are not sent to telemetry
+        ev.rh_user_has_seat = True
```

Comment:
This is not super elegant :-). I would really prefer we completely remove the prompt/repsonse from the original payload. Is this an option?